src/components/App/components/Preview/Preview.js

```diff
@@ -2,23 +2,27 @@ import styles from './Preview.less';
 
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { MAX_FONT_SIZE_IN_PX } from 'constants';
 import { str2sixDigitHex, contrast, findClosestAccessibleColor } from 'utils/color/color';
 import MultilineEllipsis from 'MultilineEllipsis/MultilineEllipsis';
 import { fontSizeInPx } from 'utils/accessibility/accessibility';
 
 const loremIpsum = `
-  Lorem ipsum dolor sit amet, ut pri essent facilis constituto, etiam assueverit
-  signiferumque ex ius. Quas quaestio ea duo. Purto magna aperiam no pri. Pri
-  prompta partiendo efficiendi ne, sed tritani deterruisset necessitatibus id,
-  ad est sint noluisse.
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 `;
 
 function calcContrast(color1, color2) {
   return Math.round(100 * contrast(color1, color2)) / 100;
 }
 
 function calcLinesToShow(fontSize) {
```

Comment:
Nope. I was referring to the parameter of the function `calcLinesToShow()`. So, just this function, not the whole file.